Description: Revert http://git.gnome.org/browse/glib/commit/?id=6560b3. Final freeze is not the time to start fixing the gsettings paths of all packages.
Author: Martin Pitt <martin.pitt@ubuntu.com>

Index: glib2.0-2.32.1/gio/glib-compile-schemas.c
===================================================================
--- glib2.0-2.32.1.orig/gio/glib-compile-schemas.c	2012-04-13 15:39:41.000000000 +0200
+++ glib2.0-2.32.1/gio/glib-compile-schemas.c	2012-04-16 15:18:07.121852684 +0200
@@ -1204,12 +1204,6 @@
       return;
     }
 
-  if (path && (g_str_has_prefix (path, "/apps/") ||
-               g_str_has_prefix (path, "/desktop/") ||
-               g_str_has_prefix (path, "/system/")))
-    g_printerr ("warning: Schema '%s' has path '%s'.  Paths starting with "
-                "'/apps/', '/desktop/' or '/system/' are deprecated.\n", id, path);
-
   state->schema_state = schema_state_new (path, gettext_domain,
                                           extends, extends_name, list_of);
 
